apps/web/components/AboutLatestNews/AboutLatestNews.tsx

```diff
@@ -61,7 +61,7 @@ export const AboutLatestNews: FC<LatestNewsProps> = ({
         {first && (
           <BigNewsItem
             news={first}
-            {...linkResolver('news', [first.slug])}
+            href={linkResolver('news', [first.slug]).href}
```

Comment:
We might also write all instances where the prop is `href` like this
```suggestion
            {...linkResolver('news', [first.slug])}
```